### ISSUE TITLE
LAVIS

### ISSUE THREAD

**Issue by StrongTanisha** [StrongResearch/isc-demos#56]
### Source / repo
https://github.com/salesforce/LAVIS

### Model description
[DESCRIPTION]

### Dataset
[DATASET]

### Literature benchmark source
[URL]

### Literature benchmark performance
[DESCRIPTION]
[VALUE/S]

### Strong Compute result achieved
[VALUE/S]

### Basic training config (as applicable)
Nodes: [N]
Epochs: [N]
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

### Logs gist
[URL]

**Comment by StrongTanisha** [comment]
LAVIS - base model no implementable - you do have to select one

All similar, all implement base class. Some don't fit into 24GB GPU (so Calvin helping with sharding)

Blip - (Blip 70BN parameters, too big for GPU). Blip 1 can work though. Lachlan working on that now

**Comment by StrongTanisha** [comment]
Now cycling. Checking on validating a full cycle, but looking promising. Next is to establish a benchmark and assess performance

**Comment by StrongTanisha** [comment]
* For the initial demo for blip, the way they evaluate uses the wrong thing - supposed to use a base class (base task) but it's using something else
* Making changes to the captioning task instead (to then make use of the base class)
* Currently on 2 nodes on staging
